### ISSUE TITLE
Different scores from original paper

### ISSUE THREAD

**Issue by nikkon3** [littleredxh/HardNegative#4]
in these links( https://arxiv.org/pdf/2007.12749.pdf, https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123590120.pdf), Table 1 has different scores. Could you explain me why ?

**Comment by littleredxh** [comment]
We slightly optimized the training code for some corner cases in the loss module(line110 to 120) and got improvement after submission.
